platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/crypto/CryptoStatic.java

```diff
@@ -326,23 +292,18 @@ private static String getPrivateKeysFileName(final String memberName) {
      * those files fails to load properly, then it returns null. It does NOT return a partial list with the
      * ones that worked.
      *
-     * @param addressBook
-     * 		the address book of the network
-     * @param keysDirPath
-     * 		the key directory, such as /user/test/sdk/data/key/
-     * @param password
-     * 		the password used to protect the PKCS12 key stores containing the node RSA public/private key pairs
-     * @return array of key stores
-     * @throws KeyStoreException
-     * 		if there is no provider that supports {@link CryptoConstants#KEYSTORE_TYPE}
-     * @throws KeyLoadingException
-     * 		in an issue occurs while loading keys and certificates
+     * @param addressBook the address book of the network
+     * @param keysDirPath the key directory, such as /user/test/sdk/data/key/
+     * @param password    the password used to protect the PKCS12 key stores containing the node RSA public/private key pairs
+     * @return map of key stores
+     * @throws KeyStoreException   if there is no provider that supports {@link CryptoConstants#KEYSTORE_TYPE}
+     * @throws KeyLoadingException in an issue occurs while loading keys and certificates
      */
     @NonNull
     static Map<NodeId, KeysAndCerts> loadKeysAndCerts(
             @NonNull final AddressBook addressBook, @NonNull final Path keysDirPath, @NonNull final char[] password)
             throws KeyStoreException, KeyLoadingException, UnrecoverableKeyException, NoSuchAlgorithmException {
-        Objects.requireNonNull(addressBook, "addressBook must not be null");
+        Objects.requireNonNull(addressBook, ADDRESS_BOOK_MUST_NOT_BE_NULL);
```

Comment:
I'm wondering about the purpose of making this particular string a string-constant but not the other strings in the `Objects.requireNonNull`.  At first I was putting these strings in as the second parameter, but there is so little value in the strings that I have been leaving them off.   But as is, there is an asymmetry of treatment of the strings in this file.

Reply:
@edward-swirldslabs This was the only constant value which was repeated in multiple places.

Reply:
IMHO, The second parameter is useful for at a glance determining which parameter was `null` without having to pull up the code and match line numbers to the exception stack trace.